dp-cluster-setup-utility.py

```diff
@@ -596,7 +607,7 @@ def __init__(self, url, credentials):
     self.client = RestClient.forJsonApi(self.base_url, credentials)
     self.available_apps = [
       DpApp('Data Steward Studio (DSS)', 'dss', dependencies=[KNOX, RANGER, DPPROFILER, ATLAS]),
-      DpApp('Data Lifecycle Manager (DLM)', 'dlm', dependencies=[KNOX, RANGER, BEACON, HIVE, HDFS]),
+      DpApp('Data Lifecycle Manager (DLM)', 'dlm', dependencies=[KNOX, RANGER, BEACON, HIVE, HDFS, ATLAS]),
```

Comment:
Is Atlas now a mandatory dependency for DLM - because this addition will now prevent the script from proceeding if Atlas is not found

Reply:
No ATLAS is not mandatory for DLM but i can't figure out any way wherein we can provide optional dependency in this script. Wanted to discuss the approach for supporting optional dependency.

Reply:
Will do in next patch.